packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -64,9 +60,7 @@ class CodeSizeBodyState extends State<CodeSizeBody>
   void initState() {
     super.initState();
     _tabController = TabController(length: tabs.length, vsync: this)
```

Comment:
wrap this in `addAutoDisposeListener`
Example: https://github.com/flutter/devtools/pull/2197

Reply:
Done